### ISSUE TITLE
HB.purchase_time field not uploading when using CSV import

### ISSUE THREAD

**Issue by JakeD22** [hay-kot/homebox#383]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### Homebox Version

0.9.0

### What is the issue you are experiencing?

I have nearly 100 items in a CSV template and the HB.purchase_time field has the date in DD/MM/YYYY but when i import that date isnt taken across on the web UI.

P.S. - I really love this project and cant wait for it to come out of beta. I came from Snipe-IT and I agree 100% with what you said about it and that how i found Homebox. keep up the good work i am a big fan. Thank you.

### How can the maintainer reproduce the issue?

Try importing a new item as CSV and add the date like 23/10/2022.

### Deployment

Unraid

**Comment by hay-kot** [comment]
Thanks for the kind words!

> Try importing a new item as CSV and add the date like 23/10/2022.

Does it work if you use YYYY-MM-DD? That's the format listed in the docs, but, I don't remember if I added additional formats to the parser.

**Comment by JakeD22** [comment]
appreciate the prompt response, I have messed with the import a few ways and no matter the format I cant get it to work for me. Something interesting is about 5 out of 96 worked. 

What I did was import all 96 with the date then exported shortly after and only 5 dates survived if that makes sense. but they were all the same format, I tried both YYYY.MM.DD & the reverse. 

I also didn't realise it was supposed to be purchase_at - what I did was make an asset manually then export it as a way to save time and get the correct headings automatically and the purchase date field came out as purchase_time if I am not mistaken. I tried your suggestion with purchase_at and even tried purchase_date. I couldn't get either to work.

another thing I noticed is the notification of a successful import at the top right stopped showing after the first few imports it did show that's how I know it exists. I have to wait and guess it has imported. some changes go through so it must be working, maybe that's my issue its not fully importing.

**Comment by hay-kot** [comment]
> I also didn't realise it was supposed to be purchase_at - what I did was make an asset manually then export it as a way to save time and get the correct headings automatically and the purchase date field came out as purchase_time if I am not mistaken. I tried your suggestion with purchase_at and even tried purchase_date. I couldn't get either to work.

You're right actually it should be 'purchase_time', this is wrong in the docs I just took a look at the code, I'll need to get that updated. 

---

Would you mind sending me a copy of the CSV so I can have a look and try to reproduce it? You can send it to me on discord, or email me at hayden@haybytes.com if you don't want to share it here publicly.

**Comment by JakeD22** [comment]
Sure thing I will email it through shortly, here is the unRAID log which is how I gauge the import has finished 

INF ../go/src/app/internal/web/mid/logger.go:30 > request finished method=POST path=/api/v1/items/import rid=ea997a573686/HFO6uPu9Gb-000931 status=204

**Comment by hay-kot** [comment]
I took a look at the import you sent me, the column for `purchase_time` is not formatted correctly. I think this may be due to a problem with how Excel or Sheets displays things to the user. If I opened up the CSV in excel I got a different text view than what I got opening it with a text editor which may explain some of the confusion in formats

Either way, This Commit https://github.com/hay-kot/homebox/pull/384/commits/3ed72daa64c77bc0ad2430257c1d0315ff1f0ecd in #384 will add support for dates formatted as YYYY/MM/DD so that should solve this problem for you, I'll also get the column name updated in the docs.

Thanks for the reproducible example!
